src/components/Timeline.jsx

```diff
@@ -0,0 +1,116 @@
+import { VerticalTimeline, VerticalTimelineElement }  from 'react-vertical-timeline-component';
+import 'react-vertical-timeline-component/style.min.css';
+import "./Timeline.css"
+import FlagIcon from '@mui/icons-material/Flag';
+
+function Timeline(){
+    return(
+<VerticalTimeline>
+  <VerticalTimelineElement
```

Comment:
code from line 9 to line 22 is getting repeated again and right so it will be great if you make a component for it and pass the text to show as props in them
This will help in reducing code repeation
Also create a timeline folder in the component folder and add the files in that